django-backend/fecfiler/committee_accounts/views.py

```diff
@@ -239,7 +241,7 @@ def remove_member(self, request, pk: UUID):
         try:
             member.delete()
             logger.info(
-                f"{request.user.email} removed {member.email} "
+                f"{request.user.id} removed {member.id} "
```

Comment:
I think `member.id` needs to be `member.user.id`.  

Just a thought, not necessary but, might be worth considering renaming the local variable `member` to `membership` if it's getting confused with representing a user

Reply:
Oh, that's a great catch!  On it~